test/Error/aggregate_error.js

```diff
@@ -0,0 +1,456 @@
+    },
+    {
+        name: "newtarget proto custom",
+        body: function ()
+        {
+            var custom = { x: 42 };
+            var newt = new Proxy(function () { }, {
+                get(t, p) {
+                    if (p === "prototype") {
+                        return custom;
+                    }
+
+                    return t[p];
+                }
+            });
+
+            var obj = Reflect.construct(AggregateError, [[]], newt);
+            assert.areEqual(custom, Object.getPrototypeOf(obj));
+            assert.areEqual(42, obj.x);
+        }
+    },
+    {
+        name: "newtarget proto fallback",
+        body: function ()
+        {
+            const values = [
+                undefined,
+                null,
+                42,
+                false,
+                true,
+                Symbol(),
+                "string",
+                AggregateError.prototype,
+            ];
+
+            const NewTarget = new Function();
+
+            for (const value of values) {
+                const NewTargetProxy = new Proxy(NewTarget, {
+                    get(t, p) {
+                        if (p === "prototype") {
+                            return value;
+                        }
+                        return t[p];
+                    }
+                });
+
+                const error = Reflect.construct(AggregateError, [[]], NewTargetProxy);
+                assert.areEqual(AggregateError.prototype, Object.getPrototypeOf(error));
+            }
+        }
+    },
+    {
+        name: "newtarget proto",
+        body: function ()
+        {
+            var obj = new AggregateError([]);
+            assert.areEqual(AggregateError.prototype, Object.getPrototypeOf(obj));
+        }
+    },
+    {
+        name: "prop desc",
+        body: function ()
+        {
+            assert.areEqual("function", typeof AggregateError);
+        }
+    },
+    {
+        name: "proto",
+        body: function ()
+        {
+            var proto = Object.getPrototypeOf(AggregateError);
+            assert.areEqual(Error, proto);
+        }
+    },
+    {
+        name: "order of args evaluation",
+        body: function () {
+            let sequence = [];
+            const message = {
+                toString() {
+                    sequence.push(1);
+                    return "";
+                }
+            };
+            const errors = {
+                [Symbol.iterator]() {
+                    sequence.push(2);
+                    return {
+                        next() {
+                            sequence.push(3);
+                            return {
+                                done: true
+                            };
+                        }
+                    };
+                }
+            };
+
+            new AggregateError(errors, message);
+            assert.areEqual(3, sequence.length);
+            assert.areEqual([1, 2, 3], sequence);
+        }
+    }

```

Comment:
Do you have cases for the treatment of the `errors` parameter? what order elements in th eprovided iterator are accessed etc.?

Valid (and invalid) inputs, no inputs at all?

Reply:
Seems Invalid case covered at `errors iterable to list failures`. And the valid case (And order) in this case.